### DISCUSSION TITLE
Light theme for official website

### DISCUSSION THREAD

**Discussion by devope** [livewire/livewire#7971] [3. Feature ideas]
Hi there
I couldn't find any theme switcher
Can not use dark mode

**Comment by devope** [comment]
Btw now I must use "Auto Invert!" Chrome extension to read docs comfortably.

**Comment by devope** [comment]
Hi there @calebporzio
Are you planning to introduce a light theme?

**Comment by troccoli-sgr** [comment]
@devope 

Add you vote and comment on #7199
